### ISSUE TITLE
Errors with no line numbers

### ISSUE THREAD

**Issue by tlovett1** [alexa-js/alexa-app-server#70]
I'm getting this in the console:

```
Error loading app [/Users/tlovett1/projects/alexa-app-server/examples/apps/airportinfo/index.js]: SyntaxError: Invalid or unexpected token
```

How can I find the line number?

**Comment by dblock** [comment]
This seems like a problem in your app, not sure why the line number is not reported. If you put the project up I can try help isolate. Also do try against HEAD here, we're about to release v. next and there're a bunch of fixes.

**Comment by dblock** [comment]
Ignore what I previously said, this is a bug. I haven't looked for a fix (yet).

**Comment by tejashah88** [comment]
Is there a way to reproduce this bug?

**Comment by dblock** [comment]
@tejashah88 Yes, make some kind of syntax error in the .js of an alexa app and run the server.

**Comment by ericblade** [comment]
You can run the skill on it's own (node index.js from the app dir) to diagnose.  Still worthy of a fix if possible, IMO, good to see where the errors occur without having to go run the app on it's own.

**Comment by dblock** [comment]
I'd love a PR please!
